platform/pulp/platform/fields.py

```diff
@@ -0,0 +1,96 @@
+"""
+Custom django fields provided by pulp platform
+"""
+import json
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import six, ugettext_lazy as _
+
+
+def _parse_json(json_string):
```

Comment:
Good point.  The development was based off of 

https://docs.djangoproject.com/en/1.8/howto/custom-model-fields/#converting-values-to-python-objects

Which has separate code in the functions to catch different scenarios.  I originally had followed something similar, while now both functions are identical.  Should be no problem to move it into to_python, and we can always move it back out if needed.